sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -593,367 +586,529 @@ pi_result piContextCreate(const pi_context_properties *Properties,
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piProgramGetBuildInfo(pi_program, pi_device, cl_program_build_info,
                                 size_t, void *, size_t *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piProgramRetain(pi_program) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piProgramRelease(pi_program) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piextProgramGetNativeHandle(pi_program, pi_native_handle *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piextProgramCreateWithNativeHandle(pi_native_handle, pi_context,
                                              pi_program *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piKernelCreate(pi_program, const char *, pi_kernel *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piKernelSetArg(pi_kernel, pi_uint32, size_t, const void *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piextKernelSetArgMemObj(pi_kernel, pi_uint32, const pi_mem *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 // Special version of piKernelSetArg to accept pi_sampler.
 pi_result piextKernelSetArgSampler(pi_kernel, pi_uint32, const pi_sampler *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piKernelGetInfo(pi_kernel, pi_kernel_info, size_t, void *, size_t *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piKernelGetGroupInfo(pi_kernel, pi_device, pi_kernel_group_info,
                                size_t, void *, size_t *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
-pi_result
-piKernelGetSubGroupInfo(pi_kernel, pi_device,
-                        pi_kernel_sub_group_info, // TODO: untie from OpenCL
-                        size_t, const void *, size_t, void *, size_t *) {
+pi_result piKernelGetSubGroupInfo(pi_kernel, pi_device,
+                                  pi_kernel_sub_group_info, size_t,
+                                  const void *, size_t, void *, size_t *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piKernelRetain(pi_kernel) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piKernelRelease(pi_kernel) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piEventCreate(pi_context, pi_event *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piEventGetInfo(pi_event, pi_event_info, size_t, void *, size_t *) {
   DIE_NO_IMPLEMENTATION;
-  return PI_SUCCESS;
 }
 
 pi_result piEventGetProfilingInfo(pi_event Event, pi_profiling_info ParamName,
                                   size_t ParamValueSize, void *ParamValue,
                                   size_t *ParamValueSizeRet) {
-  (void)Event;
-  (void)ParamName;
-  (void)ParamValueSize;
-  (void)ParamValue;
-  (void)ParamValueSizeRet;
-  DIE_NO_IMPLEMENTATION;
+  std::cerr << "Warning : Profiling Not supported under PI_ESIMD_CPU"
```

Comment:
Please, print if PI_TRACE is enabled only.

Reply:
Will revise.